src/screens/Learn/Glossary/Glossary.tsx

```diff
@@ -31,6 +32,27 @@ function getGlossaryFooter(): React.ReactElement {
   );
 }
 
+function useScrollToTerm(termToScrollTo: string) {
+  // Only scroll to the term when the page first loads.
+  const [scroll, setScroll] = useState(false);
+  useEffect(() => {
+    const element = document.querySelector(termToScrollTo);
+    if (element && !scroll) {
+      setScroll(true);
+      element.scrollIntoView();
```

Comment:
we need to account for the height of the navbar when scrolling to a location on the page. right now, the page scrolls to the correct glossary term, but part of the content is covered by the navbar (image below, you'll notice that the term header **Presymptomatic spread*** isn't visible). replacing `element.scrollIntoView()` with [scrollWithOffset](https://github.com/covid-projections/covid-projections/blob/develop/src/components/TableOfContents/utils.ts) would solve for that

<img width="1665" alt="Screen Shot 2021-01-31 at 9 06 05 PM" src="https://user-images.githubusercontent.com/44076375/106406706-251b6380-6408-11eb-978c-56c6e2044e48.png">